Domains/4-Application/OpenSite/Released/OpenSite.01.00.11.ecschema.xml

```diff
@@ -0,0 +1,1490 @@
+        <ECProperty propertyName="SidewalkOffset" typeName="double" kindOfQuantity="rru:LENGTH" category="EdgeSidewalkAspect_Sidewalk" displayLabel="Offset" description="Offset distance for sidewalk." />
+        <ECProperty propertyName="SidewalkDepth" typeName="double" kindOfQuantity="rru:LENGTH" category="EdgeSidewalkAspect_Sidewalk" displayLabel="Depth" description="Depth of sidewalk material." />
+        <ECProperty propertyName="MinSidewalkSlope" typeName="double" kindOfQuantity="rru:SLOPE" category="EdgeSidewalkAspect_Sidewalk" displayLabel="Min. Slope" description="Minimum allowable sidewalk slope." />
+        <ECProperty propertyName="MaxSidewalkSlope" typeName="double" kindOfQuantity="rru:SLOPE" category="EdgeSidewalkAspect_Sidewalk" displayLabel="Max. Slope" description="Maximum allowable sidewalk slope." />
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="BuildingEdgeOwnsSidewalkAspect" modifier="Sealed" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="false">
+            <Class class="BuildingEdge" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="EdgeSidewalkAspect" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="LayoutParkingEdgeOwnsSidewalkAspect" modifier="Sealed" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="false">
+            <Class class="LayoutParkingEdge" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="EdgeSidewalkAspect" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="DrivewayPathFinOwnsSidewalkAspect" modifier="Sealed" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="false">
+            <Class class="DrivewayPathFin" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="EdgeSidewalkAspect" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="EdgeCurbAspect" modifier="Sealed" displayLabel="Curb">
+        <BaseClass>bis:ElementUniqueAspect</BaseClass>
+
+        <ECProperty propertyName="CurbHeight" typeName="double" kindOfQuantity="rru:LENGTH" category="EdgeCurbAspect_Curb" displayLabel="Curb Height" description="Height of the curb.">
+            <ECCustomAttributes>
+                <HiddenProperty xmlns="CoreCustomAttributes.01.00.03" />
+                <Deprecated xmlns="CoreCustomAttributes.01.00.03">
+                    <Description> We handle curbs in the OpenSite+ application now. </Description>
```

Comment:
Being an outsider, this description doesn't point me in the direction of how curbs should be understood after this schema-change. It seems the authoring classes now capture "CurbTypes", so maybe the description should point readers to the introduction of "CurbType" properties to classes like "ParkingArea", "DrivewayArea" and "IslandArea".